docs/reference/buildx_eval_build.md

```diff
@@ -0,0 +1,56 @@
+| `-q`, `--quiet`                                                                                                                                    |               |           | Suppress the build output and print image ID on success                                             |
+| `--root`                                                                                                                                           | `string`      |           | Specify root directory of server to connect (EXPERIMENTAL)                                          |
+| `--sbom`                                                                                                                                           | `string`      |           | Shorthand for `--attest=type=sbom`                                                                  |
+| `--secret`                                                                                                                                         | `stringArray` |           | Secret to expose to the build (format: `id=mysecret[,src=/local/secret]`)                           |
+| `--server-config`                                                                                                                                  | `string`      |           | Specify buildx server config file (used only when launching new server) (EXPERIMENTAL)              |
+| `--shm-size`                                                                                                                                       | `bytes`       | `0`       | Shared memory size for build containers                                                             |
+| `--ssh`                                                                                                                                            | `stringArray` |           | SSH agent socket or keys to expose to the build (format: `default\|<id>[=<socket>\|<key>[,<key>]]`) |
+| [`-t`](https://docs.docker.com/reference/cli/docker/image/build/#tag), [`--tag`](https://docs.docker.com/reference/cli/docker/image/build/#tag)    | `stringArray` |           | Name and optionally a tag (format: `name:tag`)                                                      |
+| [`--target`](https://docs.docker.com/reference/cli/docker/image/build/#target)                                                                     | `string`      |           | Set the target build stage to build                                                                 |
+| `--ulimit`                                                                                                                                         | `ulimit`      |           | Ulimit options                                                                                      |
+
+
+<!---MARKER_GEN_END-->
+
+## Description
+
+Same as [`build`](buildx_build.md), but evaluate the build configuration without
+building the image.
```

Comment:
@dvdksn Could add some docs below this description with some quick examples maybe? Similar to ones use in our integration tests: https://github.com/docker/buildx/pull/2494/files#diff-9a2da9eceb368adfae46838969a2274770a558a410fa374707616b0c5737d9b6R21

Reply:
Yes, below this we would need to have at least three examples, one each for `lint`, `targets`, and `outline`. We should also link out to the lint rules reference (from the lint example).

Additionally, if we are going with this approach, we need to add a new page under the manuals section, maybe under buildkit or dockerfile, that describes build request vs subrequests, and how subrequests can be implemented in frontends to support alternate strategies for a build invocation.